pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -1286,16 +1288,18 @@ public bool ImportingSearch
             (c, p) => c.AssociateProteinsMinPeptidesPerProtein = p.ValueInt) { WrapValue = true };
 
         private static readonly ArgumentGroup GROUP_ASSOCIATE_PROTEINS = new ArgumentGroup(() => Resources.CommandLine_AssociateProteins_Associating_peptides_with_proteins, false,
-            ARG_AP_GROUP_PROTEINS, ARG_AP_SHARED_PEPTIDES, ARG_AP_MINIMAL_LIST, ARG_AP_MIN_PEPTIDES, ARG_AP_REMOVE_SUBSETS)
+            ARG_AP_GROUP_PROTEINS, ARG_AP_GENE_LEVEL, ARG_AP_SHARED_PEPTIDES, ARG_AP_MINIMAL_LIST, ARG_AP_MIN_PEPTIDES, ARG_AP_REMOVE_SUBSETS)
             { LeftColumnWidth = 45 };
 
         public bool? AssociateProteinsGroupProteins { get; private set; }
+        public bool? AssociateProteinsGeneLevelParsimony { get; private set; }
```

Comment:
I think if you add "[DefaultValuesFalse]" above this property definition, the Audit Log won't write it out when it is equal to false, so you won't need to check in new copies of the tutorial log files in this pull request.

Reply:
I think I was copying what I did for Group proteins, where I wanted it to show false even though that's the default (so users don't have to think about whether grouping proteins is the default behavior or not). But that makes less sense for gene level parsimony since it should be pretty obvious to users that the default will be by protein. So yeah, I may get to undo all those audit log changes...